mautic/mautic
# Fix timezone on timeline

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     |  Fixes #10892




#### Description:
We got on timeline this 

![image](https://user-images.githubusercontent.com/462477/131103270-e89967d3-ceff-4c7c-a6c6-20ceecbd2dd2.png)

You can see  contact created event as today, but others events happen yesterday. 
This issue was with reset timezone to UTC in  toUtcString method of DateTimehelper.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. This is hard to reproduce, I saw it on one instance and fixed it.
3. Just code review in this case should be enough




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10375"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
